### PR TITLE
[MM-22495] fix uac name

### PR DESCRIPTION
**Pull request by Willyfrog**
Please provide the following information:

**Summary**

change how to describe the signed package into its propper order
add notifications for releases

**Issue link**

**Test Cases**

on UAC should display the right version

**Additional Notes**
waiting for circle to be able to get npm from the registry

### CODE REVIEW

**Comment by Willyfrog** [comment]
for the notifications, we have to change the webhook, but it is working, see: https://community-daily.mattermost.com/core/pl/tjpgkc6jtfg5tbd7qx9oagda1r

**Comment by Willyfrog** [comment]
@metanerd @deanwhillier this is ready for review, it now shows the version at the beginning of the message.
It is currently pointing to a [test channel](https://community-daily.mattermost.com/core/pl/nd5gpinq8pdwjnt4z4fzmeygcy) but after the merge, we can update the context in CircleCI to point to the release channel or the desktop one

File: .circleci/config.yml [hunk 1]
```diff
@@ -277,11 +303,40 @@ jobs:
...
       - check
+      - begin-notification:
+          context: desktop-notify
+          filters:
+            branches:
+              only:
+                # only for release and release candidates
+                # release-XX.YY.ZZ
+                # release-XX.YY.ZZ-rc-something
+                - /^release-\d+(\.\d+){1,2}(-rc.*)?/
```
Comment (metanerd) [inline .circleci/config.yml]: This one is very nice to know. Thank you!

**Comment by Willyfrog** [comment]
@amyblais since we already have the 4.5 branch, I think this one should be cherrypicked there, right?

**Comment by wget** [comment]
@Willyfrog Chery-picked and pushed to release-4.5 =)

**Pull request merged**